friends/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Friends from './components/Friends'
+
+class App extends Component {
```

Comment:
Refer to your React-Insta-Clone project for a possible component and data flow structure. Note how in the Insta-Clone project you set state in `App.js` and sent the post data down as props to be mapped through and displayed. The same idea can be applied here, with your axios GET call happening in your `componentDidMount()`.